submit-queue/e2e.go

```diff
@@ -30,9 +30,24 @@ import (
 	github_api "github.com/google/go-github/github"
 )
 
+// PRInfo explicitly collects the fields we wish to encode via JSON.
+type PRInfo struct {
+	Number *int    `json:"number"`
+	URL    *string `json:"html_url"`
```

Comment:
I can change it if you feel strongly, but I'm pretty sure everyone treats those as read-only; they're pointers only because they are optional.